mackerel_test.go

```diff
@@ -77,3 +80,99 @@ func TestLogger(t *testing.T) {
 		t.Errorf("verbose log should match /<api>.*OK/; but %s", s)
 	}
 }
+
+type fakePrinter struct {
+	w io.Writer
+}
+
+func (p *fakePrinter) Tracef(format string, v ...interface{}) {
+	fmt.Fprintf(p.w, format, v...)
+}
+func (p *fakePrinter) Debugf(format string, v ...interface{})    {}
+func (p *fakePrinter) Infof(format string, v ...interface{})     {}
+func (p *fakePrinter) Warningf(format string, v ...interface{})  {}
+func (p *fakePrinter) Errorf(format string, v ...interface{})    {}
+func (p *fakePrinter) Criticalf(format string, v ...interface{}) {}
```

Comment:
`PrioritizedLogger` does not requires `Criticalf` anymore.

Reply:
I removed Criticalf method.